### ISSUE TITLE
Quickstart: How to access acid-minimal-cluster (Minikube) from host?

### ISSUE THREAD

**Issue by hjacobs** [zalando-incubator/postgres-operator#292]
I just tried out the quickstart (introduced with #289), but I don't know how to access the new "acid-minimal-cluster" PostgreSQL cluster:

```
$ kubectl create -f manifests/configmap.yaml  # configuration
configmap "postgres-operator" created
$ kubectl create -f manifests/operator-service-account-rbac.yaml  # identity and permissions
serviceaccount "zalando-postgres-operator" created
clusterrole "zalando-postgres-operator" created
clusterrolebinding "zalando-postgres-operator" created
$ kubectl create -f manifests/postgres-operator.yaml  # deployment
deployment "postgres-operator" created
$ kubectl get pod
NAME                                 READY     STATUS    RESTARTS   AGE
postgres-operator-6474c766b4-tgp82   1/1       Running   0          8s
$ kubectl create -f manifests/minimal-postgres-manifest.yaml 
postgresql "acid-minimal-cluster" created
$ kubectl get pod
NAME                                 READY     STATUS    RESTARTS   AGE
acid-minimal-cluster-0               0/1       Pending   0          2s
postgres-operator-6474c766b4-tgp82   1/1       Running   0          17s
$ kubectl get pod
NAME                                 READY     STATUS    RESTARTS   AGE
acid-minimal-cluster-0               1/1       Running   0          20s
acid-minimal-cluster-1               1/1       Running   0          6s
postgres-operator-6474c766b4-tgp82   1/1       Running   0          35s
$ kubectl get svc
NAME                        TYPE           CLUSTER-IP       EXTERNAL-IP   PORT(S)          AGE
acid-minimal-cluster        LoadBalancer   10.102.211.199   <pending>     5432:32114/TCP   25s
acid-minimal-cluster-repl   ClusterIP      10.106.83.140    <none>        5432/TCP         25s
kubernetes                  ClusterIP      10.96.0.1        <none>        443/TCP          1m
$ minikube service acid-minimal-cluster
Waiting, endpoint for service is not ready yet...
```

The service will never become ready as it's of type `LoadBalancer`.

**Comment by Jan-M** [comment]
FYI: We are looking into this right now.

**Comment by zerg-junior** [comment]
This behavior seems to be minikube-specific and relates to the recent upgrades that made `minikube` use Kubernetes 1.10+ 

It should be fully  resolved  once a Spilo image with a [Patroni `>= v1.4.4`](https://github.com/zalando/patroni/pull/683) is out because of that:

> Kubernetes 1.10.0+ started returning `Endpoints.subsets` set to `None` instead of `[]`.

Kudos to @mgomezch for remembering this bug just in time !


There are other configuration issues here, but these I've already fixed and am currently preparing a PR

**Comment by zerg-junior** [comment]
@hjacobs Can you please confirm the updated quickstart works ? Just do

```bash
git clone https://github.com/zalando-incubator/postgres-operator.git
cd postgres-operator
git checkout --track origin/fix-conf-for-minikube
```

and then [these instruction](https://github.com/zalando-incubator/postgres-operator/blob/f6027f810c6efcf95aa27ad4331c0cd0ff1d5376/README.md#local-execution) should work.

> The service will never become ready as it's of type LoadBalancer.

Clarification: we actually need service of such type here for it to be accessible outside of `minikube`. Such services [run fine on minikube](https://github.com/kubernetes/minikube/issues/384#issuecomment-234409957).

**Comment by hjacobs** [comment]
@zerg-junior first observation, one might need to wait some seconds:

```
kubectl create -f manifests/minimal-postgres-manifest.yaml 
error: unable to recognize "manifests/minimal-postgres-manifest.yaml": no matches for acid.zalan.do/, Kind=postgresql
```

**Comment by zerg-junior** [comment]
Addressed by #301

**Comment by zanoubia** [comment]
Not only minikube. I have the same problem with a three nodes cluster ! any ideas ?
